custom/icds_reports/utils/__init__.py

```diff
@@ -1142,3 +1142,7 @@ def get_datatables_ordering_info(request):
     order_by_name_column = request.GET.get('columns[%s][data]' % order_by_number_column)
     order_dir = request.GET.get('order[0][dir]', 'asc')
     return start, length, order_by_number_column, order_by_name_column, order_dir
+
+
+def phone_number_function(x):
+    return "+{0}{1}".format('' if str(x).startswith('91') else '91', x) if x else x
```

Comment:
We already use the [phonenumberslite][1] library. You could do this more generically with
```python
    def phone_number_function(x, from_country='IN'):
        number = phonenumbers.parse(x, from_country)
        return phonenumbers.format_number(number, phonenumbers.PhoneNumberFormat.INTERNATIONAL)
```

  [1]: https://github.com/daviddrysdale/python-phonenumbers

Reply:
(That's not to say that you **should**. Use it / don't use it.)